hpke/hpke.go

```diff
@@ -67,20 +67,19 @@ const (
 
 // Suite is an HPKE cipher suite consisting of a KEM, KDF, and AEAD algorithm.
 type Suite struct {
-	KemID  KemID
-	KdfID  KdfID
-	AeadID AeadID
-	_      struct{}
+	kemID  KEM
```

Comment:
By making these private we make it impossible to make suites. Are we sure programs won't ever want to look inside to see which KEM and KDF and AEAD are used?

Reply:
`NewSuite` is the way to construct always valid suites. 

> Are we sure programs won't ever want to look inside to see which KEM and KDF and AEAD are used?

what about a function`(Suite) Params() ` that returns a triple of algorithms.